dummy_app.py

```diff
@@ -29,14 +29,25 @@
 # write query info
 if query_example or query_file:
     st.write("Your query spectrum id: {}".format(
-            query_spectrums[0].metadata.get("spectrum_id")))
+        query_spectrums[0].metadata.get("spectrum_id")))
     fig = query_spectrums[0].plot()
     st.pyplot(fig)
 
 # load library file in sidebar
 library_file = st.sidebar.file_uploader("Choose a spectra library file...",
                                         type=['json', 'txt'])
-if library_file is not None:
+# gather default libraries
+test_library_file = os.path.join(os.path.dirname(__file__), 'tests',
+                                 'testspectrum_library.json')
+example_libs_dict = {'testspectrum_query.json': test_library_file}
```

Comment:
That should be the ``_library`` one, right?

Reply:
Yes good catch. I'll correct it.